python/models/simple_tooltip_model.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2016 Shotgun Software Inc.
+# 
+# CONFIDENTIAL AND PROPRIETARY
+# 
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit 
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your 
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights 
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+import sgtk
+
+shotgun_model = sgtk.platform.import_framework(
+    "tk-framework-shotgunutils",
+    "shotgun_model",
+)
+
+class SimpleTooltipModel(shotgun_model.SimpleShotgunModel):
```

Comment:
- Some info explaining how this is being used and when it is useful would be great
- please add to sphix
- Should this be in shotgun utils, alongside the simple shotgun model?